docs/PacketFence_Checkpoint_Quick_Install_Guide.asciidoc

```diff
@@ -0,0 +1,91 @@
+Checkpoint firewall Quick Integration Guide
+==========================================
+:encoding: UTF-8
+:lang: en
+:doctype: book
+
+include::includes/global-attributes.asciidoc[]
+
+About this Guide
+----------------
+This guide has been created in order to help sales engineers, product managers, or network specialists demonstrate the PacketFence capabilities on-site with an existing or potential customer.  It can also provide guidelines to setup a proof of concept for a potential PacketFence deployment using the *Checkpoint firewall*.
+
+Assumptions
+-----------
+* You have a configured PacketFence environment with working test equipment;
+* You have a Checkpoint firewall.
+
+
+Quick installation
+------------------
+Step 1: Enabling Identity Awareness on the Security Gateway
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To enable Identity Awareness:
+
+1. Log in to 'SmartDashboard'.
+2. From the 'Network Objects tree', expand the 'Check Point branch'.
+3. Double-click the 'Security Gateway' on which to enable 'Identity Awareness'.
+4. In the 'Software Blades' section, select 'Identity Awareness' on the 'Network Security tab'.
+The 'Identity Awareness Configuration' wizard opens.
+5. Select 'one or more options'. These options set the methods for acquiring identities of managed and
+unmanaged assets.
+6. Select 'AD Query - Lets the Security Gateway seamlessly identify Active Directory users and computers' and click Next.
```

Comment:
https://sc1.checkpoint.com/documents/R77/CP_R77_IdentityAwareness_WebAdminGuide/62050.htm

From the doc AD Query - Lets the Security Gateway seamlessly identify Active Directory users and computers